src/main/resources/META-INF/rewrite/jmockit.yml

```diff
@@ -36,6 +36,9 @@ recipeList:
   - org.openrewrite.java.ChangeType:
       oldFullyQualifiedTypeName: mockit.integration.junit5.JMockitExtension
       newFullyQualifiedTypeName: org.mockito.junit.jupiter.MockitoExtension
+  - org.openrewrite.java.ChangeType:
+      oldFullyQualifiedTypeName: mockit.integration.junit4.JMockit
```

Comment:
I think this fqn is wrong, isn't it `JUnitRunner`?

Reply:
Luckily we support [using multiple versions of a library in a project](https://docs.openrewrite.org/authoring-recipes/multiple-versions), as we needed that for rewrite-spring as well. Ideally you'll want to use a single version if you can get away with it, but in the case of removed classes that's not always an option.

Reply:
Fantastic! In that case, this change should be good. I'll go ahead and update my other comment.